src/components/RegType/OnlineNoNvrf.jsx

```diff
@@ -9,6 +9,8 @@ function OnlineNoNVRF(props) {
     const stateContent = props.stateData;
     const stringContent = props.stringContent
 
+    console.log('onlinenonvrf', stateContent.name)
```

Comment:
need to remove the log statement - doesn't seem to be used to be used for any error handling right?